src/pages/well-architected-framework/index.tsx

```diff
@@ -1,48 +1,63 @@
+import slugify from 'slugify'
 import { getCollectionsBySection } from 'lib/learn-client/api/collection'
 import { stripUndefinedProperties } from 'lib/strip-undefined-props'
 import { PageSlugOption } from 'lib/learn-client/api/page'
-import { buildLayoutHeadings as buildLayoutHeadingsFromBlocks } from 'views/product-tutorials-view/helpers/heading-helpers'
 import WellArchitectedFrameworkLandingView from 'views/well-architected-framework'
-import getProductPageContent from 'views/product-tutorials-view/helpers/get-product-page-content'
-import processPageData from 'views/product-tutorials-view/helpers/process-page-data'
 import { buildCategorizedWafSidebar } from 'views/well-architected-framework/utils/generate-sidebar-items'
 import { WellArchitectedFrameworkLandingProps } from 'views/well-architected-framework/types'
-import wafContent from 'content/well-architected-framework/index.json'
+import rawWafContent from 'content/well-architected-framework/index.json'
 import wafData from 'data/well-architected-framework.json'
+import getProcessedPageData from 'views/product-tutorials-view/helpers/page-data'
+import { TableOfContentsHeading } from 'layouts/sidebar-sidecar/components/table-of-contents'
 
 export async function getStaticProps(): Promise<{
 	props: WellArchitectedFrameworkLandingProps
 }> {
-	const {
-		pageData: rawPageData,
-		inlineCollections,
-		inlineTutorials,
-	} = await getProductPageContent(wafData.slug as PageSlugOption)
-	const { pageData } = await processPageData(rawPageData)
+	const { pageData, headings: pageHeadings } = await getProcessedPageData(
+		wafData.slug as PageSlugOption,
+		{ showOverviewHeading: false }
+	)
 	const wafCollections = await getCollectionsBySection(wafData.slug)
-	const headings = [
-		{ title: wafContent.landingPage.overview.heading, level: 2 },
-		...buildLayoutHeadingsFromBlocks({
-			...pageData,
-			showOverviewHeading: false,
-		}),
+
+	/**
+	 * Build and add the slug for the overview heading
+	 */
+	const wafContent = {
```

Comment:
Bit of a convoluted object clone here, but wanted to make sure the `headingSlug` is in place, so that this heading functions as expected.

For context, currently on [/well-architected-framework](https://developer.hashicorp.com/well-architected-framework) the first heading in the sidecar doesn't function as expected, since previously the `slug` for the `overview` heading was `undefined`.